### ISSUE TITLE
Train Codes

### ISSUE THREAD

**Issue by Royalvice** [VAST-AI-Research/TriplaneGaussian#4]
What a wonderful work!
Are you plan to release your training codes?

Best Wishes!

**Comment by ddl-donglin** [comment]
At least show the part of the loss function would be very appreciated. 👍

**Comment by hadillivvy** [comment]
+1

**Comment by kihoon96** [comment]
+2

**Comment by LiyCg** [comment]
+3

**Comment by yangzhou24** [comment]
+4

**Comment by zouzx** [comment]
Hi, we don't have plans to release the training codes recently, but we will discuss it later. The loss functions are all commonly used; for example, you can find the implementation of 3D loss in [Snowflake](https://github.com/AllenXiangX/SnowflakeNet), LPIPS loss in [PerceptualSimilarity](https://github.com/richzhang/PerceptualSimilarity) and SSIM loss in [pytorch-msssim](https://github.com/VainF/pytorch-msssim). If you have any other questions about training details, please feel free to contact us, and we will provide the details as much as we can.

**Comment by SerdarHelli** [comment]
If we have train codes , we can train or we can try with new datasets. Its going to help also to get reference etc.

**Comment by StarsTesla** [comment]
Yeah, training on one dataset are very limit, like cars, trees, and other realistic things in our life is very important to not only on research field, so we want expand it, fit it to our data.

**Comment by hiyyg** [comment]
+1
